src/common/daft-config/src/lib.rs

```diff
@@ -74,6 +75,7 @@ pub struct DaftExecutionConfig {
     pub native_parquet_writer: bool,
     pub use_experimental_distributed_engine: bool,
     pub min_cpu_per_task: f64,
+    pub suggested_batch_size: u64,
```

Comment:
There's a convention of using `usize`, but `usize` is platform dependent. We should instead use a platform independent unsigned int.